shared/common-adapters/avatar.desktop.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import React, {Component} from 'react'
+import resolveRoot from '../../desktop/resolve-root'
+import type {Props} from './avatar'
+
+const noAvatar = `file:///${resolveRoot('shared/images/no-avatar@2x.png')}`
+
+export default class Avatar extends Component {
+  props: Props;
+
+  render () {
+    return (
+      <img
+        style={{width: this.props.size, height: this.props.size, borderRadius: this.props.size / 2, ...this.props.style}}
+        src={this.props.url}
+        onError={event => (event.target.src = noAvatar)}/>)
```

Comment:
also it doesn't seem like we can get any real info from the error so we can't differentiate why its broken